unipipe/main.ts

```diff
@@ -19,4 +19,7 @@ registerTransformCmd(program);
 registerUpdateCmd(program);
 registerGenerateCmd(program);
 
-await program.parse(Deno.args);
+if (Deno.args.length > 0)
```

Comment:
s: with the commit description it's clear what it does, but if you could add a tiny variable it would preserve that intent also in the code

```
const hasCmd = Deno.args.length > 0
if (hasCmd) { 
 ...
} 
else {
 showHelp()
}
```